XRTK-Core/Assets/XRTK.Tests/Core/TestFixture_02_MixedRealityToolkitUtilityTests.cs

```diff
@@ -16,56 +18,52 @@ namespace XRTK.Tests.Core
 {
     public class TestFixture_02_MixedRealityToolkitUtilityTests
     {
+        private void SetupServiceLocator()
+        {
+            TestUtilities.InitializeMixedRealityToolkitScene();
+            MixedRealityToolkit.Instance.ActiveProfile.RegisteredServiceProvidersProfile = ScriptableObject.CreateInstance<MixedRealityRegisteredServiceProvidersProfile>();
+        }
+
         #region Configuration Validation Tests
 
         [Test]
         public void Test_01_ConfirmDataProviderConfigurationNotPresent()
         {
-            var dataProviderTypes = new[] { typeof(TestDataProvider1) };
-            var dataProviderConfiguration = new[] { new ControllerDataProviderConfiguration(typeof(TestDataProvider1), "Test Controller Data Provider", 2, SupportedPlatforms.WindowsStandalone | SupportedPlatforms.Editor, null) };
-
-            TestUtilities.InitializeMixedRealityToolkitScene();
-
-            InitializeDefaultInputSystemProfile();
-
-            Assert.IsFalse(ValidateConfiguration.ValidateDataProviders(dataProviderTypes, dataProviderConfiguration, false));
+            SetupServiceLocator();
```

Comment:
See comment above. Instead of running this setup yourself here, test preparation can be done using `[SetUp]`

Reply:
There is also `[TearDown]` if needed to cleanup things after a test.

Reply:
I think we should do a separate PR to review the Unit tests and add these features.  We need more unit tests too to be sure.

Reply:
Makes sense